linux/filesystem/backup.md

```diff
@@ -15,32 +15,19 @@ sudo tar czf pi_home.tar.gz pi
 
 This creates a tar archive called `pi_home.tar.gz` in `/home/`. You should copy this file to a USB stick or transfer it to another machine on your network.
 
-## MySQL
-
-If you have MySQL databases running on your Raspberry Pi, it would be wise to keep them backed up too. To back up a single database, use the `mysqldump` command:
-
-```bash
-mysqldump recipes > recipes.sql
-```
+## SD card copier (Recommended)
 
-This command will back up the `recipes` database to the file `recipes.sql`. Note that, in this case, no username and password have been supplied to the `mysqldump` command. If you don't have your MySQL credentials in a `.my.cnf` configuration file in your home folder, then supply the username and password with flags:
+The SD Card Copier application, which can be found on the `Accessories` menu of the Raspberry Pi Desktop, will copy Raspbian from one card to another. To use it, you will need a USB SD card writer.
 
-```bash
-mysqldump -uroot -ppass recipes > recipes.sql
-```
+To back up your existing Raspbian installation, put a blank SD card in your USB card writer and plug it into your Pi, and then launch SD Card Copier. In the ‘Copy From Device’ box, select the internal SD Card. This could have a number of different names, but will usually be the first item in the list, and may have something like `(/dev/mmcblk0)` in its entry. Then select the USB card writer in the ‘Copy To Device’ box (where it will probably be the only device listed). Press ‘Start’. The copy, depending on the size of the SD card, can take ten or fifteen minutes, and when complete you should have a clone of your current installation on the new SD card. You can test it by putting the newly-copied card into the Pi’s SD card slot and booting it; it should boot and look exactly the same as your original installation, with all your data and applications intact.
```

Comment:
Maybe move the "but" to the end? :peach: 

"This could have a number of different names, and may have something like `(/dev/mmcblk0)` in its entry, but will usually be the first item in the list."